190701229_calculator_c++

thuhinkhanna opened the pull request:
Function Definitions of Arithmetic Operations are Updated.

Commit:
Update Addition Function in calculator.cpp

Completed Definition of Outlined function

Definition of the function Addition has been updated along with appropriate changes in get_input()

Commit:
Update Subtraction Function in calculator.cpp

Completed Definition of One Outlined function

Definition of the function Subtraction has been updated along with appropriate changes in get_input()

Commit:
Update Multiplication Function in calculator.cpp

Completed Definition of One Outlined function

Definition of the function Multiplication has been updated along with appropriate changes in get_input()

Commit:
Update Division Function in calculator.cpp

Completed Definition of One Outlined function

Definition of the function Division has been updated along with appropriate changes in get_input()

surabhiojha12 commented on calculator.cpp:
```diff
@@ -2,26 +2,32 @@
 
 int division(int number_1, int number_2){
-
+return number_1/number_2;
```
what if divider is 0? 5/0?

surabhiojha12 commented on calculator.cpp:
```diff
@@ -2,26 +2,32 @@
+int a,b;
+cout<<"\n Enter two Integers :: ";
+cin>>a>>b;
+cout<<"\n Sum of Numbers : "<<addition(a,b);
```
Displaying the numbers as well will improve user experience.
Sum of Numbers 6 and 5 is 11. Something like this.

thuhinkhanna replied:
Thanks for the review. Will do the updates.

Commit:
Update Main Function in calculator.cpp

Updated main function

- main function modified to throw error for invalid division by zero

- improved usability

surabhiojha12 commented on calculator.cpp:
```diff
@@ -2,26 +2,35 @@
 
 int get_input(){
-
+int a,b;
```
name the variables properly.

Commit:
Update Input function in calculator.cpp

Changed Variable Names

Changed the Identifiers names to improve readability of the source code.

Commit:
Updated calculator.cpp

Changed Identifier  names to improve readability.

thuhinkhanna committed:
Delete calculator.cpp

thuhinkhanna reviewed (commented):
Accidentally Deleted calculator.cpp
pushing calculator.cpp AGAIN

Commit:
FInal calculator.cpp

Commit:
Merge branch '229_calculator' of https://github.com/surabhiojha12/Simple_Calculator into 229_calculator
